integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -940,6 +941,7 @@ public static synchronized int getNextFreePort() {
     while (port <= END_PORT) {
       freePort = port++;
       if (isLocalPortFree(freePort)) {
+        assertDoesNotThrow(() -> TimeUnit.SECONDS.sleep(3));
```

Comment:
Got a feedback from team to wait for few seconds after the socket.close() call to give enough time to release the port.